script/package

```diff
@@ -4,7 +4,7 @@ set -e
 
 version=$(git rev-parse --short HEAD)
 
-sed -i '' -e"s/google_tracking_id:.*/google_tracking_id:/" _config.yml
+sed -i'' -e"s/tracking_id\s*:.*/tracking_id:/" _config.yml
```

Comment:
You lead me on quite a regex journey @bazzaar! thank you, was so fun to do. you're right that there was a problem with the script

Besides the fact that `google_tracking_id` doesn't exist here, the change in the sed command was slightly off. i think the `\s` was throwing it off? 

```suggestion
sed -i'' -e "s/tracking_id.*/tracking_id:/" _config.yml
```
This seems to work for me, give it a try. It now takes everything after `tracking_id` (which is captured by the `.*`) and replace that whole match (`tracking_id`+ everything else) to be just `tracking_id:`.